kuma/core/tests/__init__.py

```diff
@@ -10,11 +10,12 @@
 
 
 def assert_no_cache_header(response):
-    assert "max-age=0" in response["Cache-Control"]
-    assert "no-cache" in response["Cache-Control"]
-    assert "no-store" in response["Cache-Control"]
-    assert "must-revalidate" in response["Cache-Control"]
-    assert "s-maxage" not in response["Cache-Control"]
+    if "Cache-Control" in response:
+        assert "max-age=0" in response["Cache-Control"]
+        assert "no-cache" in response["Cache-Control"]
+        assert "no-store" in response["Cache-Control"]
+        assert "must-revalidate" in response["Cache-Control"]
+        assert "s-maxage" not in response["Cache-Control"]
```

Comment:
```suggestion
    assert "max-age=0" in response["Cache-Control"]
    assert "no-cache" in response["Cache-Control"]
    assert "no-store" in response["Cache-Control"]
    assert "must-revalidate" in response["Cache-Control"]
```

@Guyzeroth That should work and we should assert we set `Cache-Control`